vega-lite-schema.json

```diff
@@ -37,6 +37,72 @@
             ],
             "type": "string"
         },
+        "AreaConfig": {
```

Comment:
I don't think we should regenerate the schema in a pr unless it is necessary so that tests pass